cps/tasks/download.py

```diff
@@ -157,7 +156,7 @@ def name(self):
         return N_("Download")
 
     def __str__(self):
-        return f"Download task for {self.media_url_link}"
+        return f"Download task for {self.media_url}"
```

Comment:
https://github.com/iiab/calibre-web/blob/0264fe555da911030adb31342fa55872895fc5c7/cps/tasks/download.py#L159C10-L159C53
```suggestion
        return f"Download task for {self.media_url}"
```